terminal/bundles/org.eclipse.terminal.view.ui/plugin.xml

```diff
@@ -309,104 +318,13 @@
-            helpContextId="org.eclipse.terminal.view.ui.command_Launch"
-            id="org.eclipse.terminal.view.ui.command.launch"
-            name="%command.launch.selection.name">
-      </command>
-    <command
-            categoryId="org.eclipse.terminal.view.ui.commands.category"
-            helpContextId="org.eclipse.terminal.view.ui.command_Launch"
-            id="org.eclipse.terminal.view.ui.command.launchConsole"
-            name="%command.launch.selection.name">
-      </command>
-      <command
-            categoryId="org.eclipse.terminal.view.ui.commands.category"
-            helpContextId="org.eclipse.terminal.view.ui.command_Launch"
-            id="org.eclipse.terminal.view.ui.command.launchToolbar"
-            name="%command.launch.name">
-      </command>
-      <command
-            categoryId="org.eclipse.terminal.view.ui.commands.category"
-            helpContextId="org.eclipse.terminal.view.ui.command_Disconnect"
-            id="org.eclipse.terminal.view.ui.command.disconnect"
-            name="%command.disconnect.name">
-      </command>
-      <command
-            categoryId="org.eclipse.terminal.view.ui.commands.category"
-            helpContextId="org.eclipse.terminal.view.ui.command_NewView"
-            id="org.eclipse.terminal.view.ui.command.newview"
-            name="%command.newview.name">
-      </command>
-   </extension>
    <extension point="org.eclipse.ui.bindings">
       <key
             commandId="org.eclipse.terminal.view.ui.command.launchToolbar"
             contextId="org.eclipse.ui.contexts.window"
             schemeId="org.eclipse.ui.defaultAcceleratorConfiguration"
             sequence="CTRL+SHIFT+M3+T"/> <!-- Bug 435111: Don't use M1 since COMMAND+Option T already taken on Mac -->
    </extension>
-   <extension point="org.eclipse.ui.handlers">
-      <handler
-            class="org.eclipse.terminal.view.ui.internal.handler.LaunchTerminalCommandHandler"
-            commandId="org.eclipse.terminal.view.ui.command.launch">
-      </handler>
-      <handler
-            class="org.eclipse.terminal.view.ui.internal.handler.LaunchTerminalCommandHandler"
-            commandId="org.eclipse.terminal.view.ui.command.launchToolbar">
-      </handler>
-      <handler
-            class="org.eclipse.terminal.view.ui.internal.handler.LaunchTerminalCommandHandler"
-            commandId="org.eclipse.terminal.view.ui.command.launchConsole">
-      </handler>
-      <handler
-            class="org.eclipse.terminal.view.ui.internal.handler.DisconnectTerminalCommandHandler"
-            commandId="org.eclipse.terminal.view.ui.command.disconnect">
-            <enabledWhen>
-               <with variable="activePart">
-                  <instanceof value="org.eclipse.terminal.view.ui.ITerminalsView"/>
-                  <test property="org.eclipse.terminal.view.ui.canDisconnect" value="true"/>
-               </with>
-            </enabledWhen>
```

Comment:
For me it looks like this should become a core expression instead of doing it "in code" @mickaelistria wdyt?

@vogella @opcoach @ruspl-afed it seems handler in E4 can currently only use annotated code based enablement, should we allow core expressions as well? If yes please add an issue so we can track missing E4 features!

Reply:
Submitted a task now to support core expressions for handlers:

- https://github.com/eclipse-platform/eclipse.platform.ui/pull/3420